src/containers/SortingView.tsx

```diff
@@ -51,47 +49,21 @@ const calculationPool = createCalculationPool({maxSimultaneous: 6})
 
 const SortingView: React.FunctionComponent<Props> = (props) => {
   const hither = useContext(HitherContext)
-  const { plugins, workspaceInfo, sorting, sortingId, recording, curationDispatch, onSetSortingInfo, onSetRecordingInfo, onSetExternalUnitMetrics } = props
+  const { plugins, workspaceInfo, sorting, sortingId, recording, curationDispatch, onSetExternalUnitMetrics } = props
   const { workspaceName, feedUri, readOnly } = workspaceInfo;
   const [externalUnitMetricsStatus, setExternalUnitMetricsStatus] = useState<CalcStatus>('waiting');
   //const initialSortingSelection: SortingSelection = {currentTimepoint: 1000, animation: {currentTimepointVelocity: 100, lastUpdateTimestamp: Number(new Date())}}
   const initialSortingSelection: SortingSelection = {}
   const [selection, selectionDispatch] = useReducer(sortingSelectionReducer, initialSortingSelection)
-  useEffect(() => {
-    if (recording?.recordingInfo) {
-      if (!selection.timeRange) {
-        const newTimeRange = {min: 0, max: Math.min(recording.recordingInfo.num_frames, Math.floor(recording.recordingInfo.sampling_frequency / 10))}
-        selectionDispatch({type: 'SetTimeRange', timeRange: newTimeRange})
-      }
-    }
-  }, [recording?.recordingInfo, selection.timeRange])
-  // const [anchorUnitId, setAnchorUnitId] = useState<number | null>(null)
   
-  // const [focusedUnitId, setFocusedUnitId] = useState<number | null>(null)
-
-  // const effect = async () => {
-  //   if ((sorting) && (recording) && (!sorting.sortingInfo) && (sortingInfoStatus === 'waiting')) {
-  //     setSortingInfoStatus('computing');
-  //     const sortingInfo = await createHitherJob(
-  //       'createjob_get_sorting_info',
-  //       { sorting_object: sorting.sortingObject, recording_object: recording.recordingObject },
-  //       { kachery_config: {}, useClientCache: true }
-  //     );
-  //     onSetSortingInfo({ sortingId, sortingInfo });
-  //     setSortingInfoStatus('finished');
-  //   }
-  // }
-  // useEffect(() => {effect()});
-
-  const {result: sortingInfo, job: sortingInfoJob} = useHitherJob<SortingInfo>(
-    'createjob_get_sorting_info',
-    { sorting_object: sorting?.sortingObject, recording_object: recording?.recordingObject },
-    { useClientCache: true }
-  )
+  const sortingInfo = useSortingInfo(sorting?.sortingObject, sorting?.recordingObject)
```

Comment:
Once again, useSortingInfo and useRecordingInfo simplify the process of getting the recording/sorting info.